catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -117,6 +117,19 @@
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.lookfirst</groupId>
+            <artifactId>sardine</artifactId>
+            <version>5.7</version>
```

Comment:
Do you have a preference as to what pom level the version property lives in? I had left it local since it was only used in one place, so my first inclination would be to put the property in this pom. That had seemed a little redundant, but I can see the reasoning from a consistency standpoint.